CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## v0.37.0 [2023-05-19]
+
+_Breaking changes_
+
+- Removed columns `private_key_data`, `private_key_type` and `public_key_data` from `gcp_service_account_key` table. ([#449](https://github.com/turbot/steampipe-plugin-gcp/pull/449))
```

Comment:
```suggestion
- Removed columns `private_key_data`, `private_key_type` and `public_key_data` from `gcp_service_account_key` table since they always returned `null`. ([#449](https://github.com/turbot/steampipe-plugin-gcp/pull/449))
```